### ISSUE TITLE
[RFC] nimble install --force

### ISSUE THREAD

**Issue by yglukhov** [nim-lang/nimble#169]
It would be nice to have a way of updating all dependencies for a project in current directory. Our project uses a lot of dependencies that are changed quite often.

**Comment by dom96** [comment]
I think that will be what ``nimble update`` will do after #158 is implemented, still haven't worked that out completely but better dependency management is definitely a must-have. #88 will also come into play here.

What I envision ``nimble install --force`` to do, is to force the installation of a Nimble package even if some dependencies don't match or there is some other problem. I think it's @Araq's dream.

**Comment by yglukhov** [comment]
Lets clarify how ``nimble update`` should work then. Here's what i think.
```sh
nimble update <pkg> # same as "nimble install -y <pkg> && for dep in deps; do nimble install -y dep"
nimble update # same as above but works only when there's nimble file in PWD, and doesn't install/update the pkg itself, only dependencies
```

**Comment by dom96** [comment]
I think ``update`` should get a ``--deps``/``--deps-only`` flag. The behaviour you are suggesting would be inconsistent.

**Comment by dom96** [comment]
Demoting this to medium priority for now. I would like to hear what you think of how Nimble now installs "special" versions of dependencies ("#head" etc).

**Comment by dom96** [comment]
@yglukhov Thoughts?

Perhaps it makes sense to implement lock files first as I think this command will play an important part in that feature.

**Comment by yglukhov** [comment]
@dom96, agreed. This should be somewhere on top of lock files.

**Comment by kobi2187** [comment]
I think nimble should update packages globally, IOW, system-wise, not folder-wise.
I don't like the npm approach. 
I think nimble already has a way to specify a particular version of a library, so I don't see the merit in doing it just for a specified directory.
